### ISSUE TITLE
Bogus error if ffmpeg can't write

### ISSUE THREAD

**Issue by glensc** [althonos/ffpb#17]
```
$ ffpb -hide_banner -i 'foo.mp4' -vf 'subtitles=filename=foo.srt:force_style=Fontsize=28' -c:a copy 'foo-cc.mp4'
Traceback (most recent call last):
  File "/usr/local/bin/ffpb", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/ffpb.py", line 173, in main
    print(notifier.lines[-1].decode(sys.stdin.encoding), file=stream)
TypeError: decode() argument 1 must be string, not None
```

The underlying problem was that FFmpeg can't write output file:

```
$ touch foo-cc.mp4
touch: cannot touch 'foo-cc.mp4': Permission denied
```

**Comment by althonos** [comment]
Good catch, this one looks like it could be prevented.

**Comment by althonos** [comment]
Should be fixed in `v0.3.0`.
